src/range.rs

```diff
@@ -2,12 +2,42 @@
 //! (`Range<T>`); this is the type for values created by a `a..b`
 //! expression. You will rarely need to interact with it directly
 //! unless you have need to name one of the iterator types.
+//! 
+//! ```
+//! use rayon::prelude::*;
+//! 
+//! let r = (0..100u64).into_par_iter()
+//!                    .fold(|| 0, |acc, cur| acc + cur)
+//!                    .sum();
```

Comment:
I notice in both examples you use `.fold(...).sum()`, when just `.sum()` should work fine.  Was this intentional?